hiring/details.rst

```diff
@@ -53,3 +35,18 @@ We are able to provide a supportive and very flexible work environment,
 with a strong sense of mission.
 All of the members of our team have worked with us for more than 2 years,
```

Comment:
Maybe this can be rephrased as "Most of the members of our team have worked with us for more than 4 years" (4 people from 6, that's it) because we hired people last year 😄